front/src/modules/pathfinding/components/Itinerary/ItineraryV2.tsx

```diff
@@ -108,16 +99,26 @@ const ItineraryV2 = ({ path }: ItineraryProps) => {
       </div>
       {displayTypeAndPath && (
         <div className="mb-2">
-          <TypeAndPath zoomToFeature={zoomToFeatureInItinerary} />
+          <TypeAndPathV2
+            zoomToFeature={zoomToFeatureInItinerary}
+            setPathProperties={setPathProperties}
+          />
         </div>
       )}
-      {origin && destination && (
+      {origin && destination && pathProperties && (
         <div className="d-flex flex-row">
-          {geojson && (
+          {pathProperties && (
```

Comment:
I think it should be like this, but maybe I misunderstood
```suggestion
          {pathProperties?.suggestedOperationalPoints && pathProperties.suggestedOperationalPoints.length  && (
```

Reply:
Yes you're right :) Not sure we need the .length test though. If it's defined, it means that pathfinding worked an we will have some ops in it.